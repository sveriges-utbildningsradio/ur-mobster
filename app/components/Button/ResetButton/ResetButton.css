.resetButton {
  align-self: center;
  background-color: #110737;
  border-radius: 10px;
  border: solid 2px #ff4a7e;
  color: #ff4a7e;
  height: 50px;
  margin-top: 35px;
  padding: 0 20px;
}

.resetButton:hover {
  border-color: #fff;
  color: #fff;
}
