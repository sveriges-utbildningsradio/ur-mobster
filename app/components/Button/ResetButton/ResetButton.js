// @flow

import React from 'react'
import { FormattedMessage } from 'react-intl'
import styles from './ResetButton.css'

type ResetButtonProps = {
  handleClick: () => void,
  id: string
}

const ResetButton = ({ handleClick, id }: ResetButtonProps) => (
  <button className={styles.resetButton} onClick={handleClick} type="button">
    <FormattedMessage id={id} />
  </button>
)

export default ResetButton
