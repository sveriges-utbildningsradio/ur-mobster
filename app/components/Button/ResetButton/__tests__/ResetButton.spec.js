import React from 'react'
import { cleanup } from '@testing-library/react'
import 'jest-dom/extend-expect'
import renderWithReactIntl from '../../../../utils/testHelpers'
import ResetButton from '../ResetButton'

describe('components/Button/ResetButton', () => {
  afterEach(cleanup)

  it('renders', () => {
    const { container } = renderWithReactIntl(
      <ResetButton handleClick={jest.fn()} id="resetBreak" />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
