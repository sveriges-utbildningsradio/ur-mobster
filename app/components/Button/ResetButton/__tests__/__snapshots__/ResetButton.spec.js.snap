// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Button/ResetButton renders 1`] = `
<button
  class="resetButton"
  data-e2e="resetBreak"
  type="button"
>
  <span>
    Avsluta rast
  </span>
</button>
`;
